tools/x-docs/src/data/docs-menu.yml

```diff
@@ -0,0 +1,22 @@
+- title: Getting started
+  items:
+    - title: Getting started
+      link: /docs
+    - title: Setup
+      link: /docs/setup
+- title: Components
+  items:
+    - title: Components
+      link: /docs/components
+    - title: Conventions
+      link: /docs/conventions
+    - title: Create a component
+      link: /docs/creating-a-component
+- title: Applications
+  items:
+    - title: Quick start
+      link: /docs/apps
+    - title: Linking components
+      link: /docs/apps/linking
+    - title: Setup
+      link: /docs/apps/setup
```

Comment:
This menus via config pattern is copied over from the Gatsby docs. Auto building the menu might be preferable as per the V1 implementation but that requires more code so 🤷‍♂️